classes/student.rb

```diff
@@ -1,9 +1,12 @@
 require_relative './person'
 
 class Student < Person
+  attr_accessor :name, :classroom
+
   def initialize(classroom, age, name = 'unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
+    @name = name
```

Comment:
- [ ] I see that you added the `classroom` as `attr_accessor`, however, this still doesn't meet the requirement pointed out by the previous reviewer, as you now have a setter for the classroom, but it doesn't add the student to the classrooms' students. 🤔 What you can do is instead of making it an `attr_accessor`, keep it as `attr_reader` and define your own setter function `def classroom=` that first, initialize the student's `classroom` attribute and second, add the student to the classrooms' students as shown in the [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) the previous reviewer shared. 🤓 🔧 

Something like this:

![image](https://user-images.githubusercontent.com/77069011/231821017-0f7c634a-11a5-4eaf-82ea-a3f062a9ed7d.png)